marlowe-dashboard-client/src/Toast/Types.purs

```diff
@@ -74,7 +72,7 @@ successToast shortDescription =
 errorToast :: String -> Maybe String -> ToastMessage
 errorToast shortDescription longDescription =
   { shortDescription
-  , longDescription
+  , longDescription: map (\m -> m <> " " <> contactSupportMessage) longDescription
```

Comment:
There are some messages that already have some sort of "contact support". If we want this for all messages we could ask russ to design something that is not just text.

Reply:
I made sure none of the other messages had any "contact support" thing at the end. I figured we'd want the same thing everywhere.